packages/json-file/src/JsonFile.ts

```diff
@@ -10,7 +10,7 @@ import { codeFrameColumns } from '@babel/code-frame';
 
 import JsonFileError from './JsonFileError';
 
-const readFileAsync = promisify(fs.readFile);
+const readFileAsync = promisify(readFile);
```

Comment:
I wish we could use `fs.promises`, but in node 10.x.x accessing the property produces a `ExperimentalWarning: The fs.promises API is experimental`.